launch.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
+
+title Bluespess
+
+node index.js
```

Comment:
Can you create a seperate `launch_debug.bat` that launches it as `node --inspect index.js` instead of just `node index.js`? The --inspect flags allows me to use Chrome DevTools on the node.js process, which is really helpful when debugging things.